packages/fast-components-react-msft/app/components/carousel-hero-content.tsx

```diff
@@ -2,21 +2,54 @@ import * as React from "react";
 import {
     CallToAction,
     CallToActionAppearance,
+    CallToActionProps,
     Heading,
+    HeadingProps,
     HeadingSize,
     HeadingTag,
     Image,
+    ImageProps,
     Paragraph,
+    ParagraphProps,
     ParagraphSize,
 } from "../../src/index";
 
+export interface CarouselHeroContentProps {
+    heading?: HeadingProps;
+    paragraph?: ParagraphProps;
+    callToAction: CallToActionProps;
+    image: ImageProps;
```

Comment:
can you either make these optional with ? or add these to required in the JSON schema.